src/Common/src/System/Security/Cryptography/DSASecurityTransforms.cs

```diff
@@ -0,0 +1,397 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Ah, right, you can't see the commit that moved it, because I squashed it out :smile:.

This is all new code, "SecurityTransforms" is one piece of Apple's crypto stack.  It's out here in Common because a) I'll need it short-term to dual-compile into X509Certificates, b) medium-to-long-term it turn into a public type in a new rich implementation type.  So it's all following the "put the guts of the type in Common and we'll dual compile it" pattern of our other two base platform implementations... just the second caller isn't visible in this PR.